lib/ramble/ramble/test/cmd/debug.py

```diff
@@ -6,16 +6,16 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
-import pytest
+# import pytest
```

Comment:
Just remove this instead of commenting it out?

Reply:
Done (I felt a bit weird not including a test framework in the test, and can only presume that should be added back in the near future..)